internal/clients/diffreviewer/circleci/circleci_service_test.go

```diff
@@ -226,6 +226,51 @@ func (c *circleCiTestSuite) TestGetDiff() {
 	c.Equal(expectedFileDiffs, fileDiffs, "invalid fileDiff return value")
 }
 
+func (c *circleCiTestSuite) TestWriteCircleComments() {
+	tp := c.initTestParams()
+	ctrl := gomock.NewController(c.T())
+	defer ctrl.Finish()
+	testCircleService := &Service{
+		Logger: circlelogger.NewDefaultCircleLogger(),
+		PrDetails: prDetails{
+			CommitSha: commitSha,
+			Owner:     testOwner,
+			Repo:      testRepo,
+		},
+	}
+	tp.cs = testCircleService
+
+	testComments, _ := makeTestGithubRepositoryComments(
+		"testComment",
+		"/comments.txt",
+		tp.cs.PrDetails.CommitSha,
+		60,
+	)
+	emptyComments := []*diffreviewer.Comment{}
+
+	tests := []struct {
+		giveComments []*diffreviewer.Comment
+		wantError    error
+		desc         string
+	}{
+		{
+			giveComments: testComments,
+			wantError:    nil,
+			desc:         "single batch comments test",
+		},
+		{
+			giveComments: emptyComments,
+			wantError:    nil,
+			desc:         "no comments test",
+		},
+	}
+
+	for _, tt := range tests {
+		err := tp.cs.WriteComments(tt.giveComments)
+		c.NoError(err, fmt.Sprintf("Error writing comments for %s test", tt.desc))
```

Comment:
yup about what brad said - we care specifically about what's being written, less so about whether or not there's an error (although obviously good to also assert that there isn't)

Reply:
also probably good to test each of the CircleServices - 1 assert that the corret mock GHComments calls are being made, the other that the logger is logging the appropriate info